extensions/ql-vscode/test/pure-tests/query-history-info.test.ts

```diff
@@ -80,4 +90,24 @@ describe('Query history info', () => {
       expect(queryText).to.equal(variantAnalysisHistoryItem.variantAnalysis.query.text);
     });
   });
+
+  describe('buildRepoLabel', () => {
+    it('should build the repo label for remote query history items', () => {
```

Comment:
Shall we cover a few more cases? E.g. remote query with no repo count, variant analysis with 1 repository.

Reply:
Added more cases in https://github.com/github/vscode-codeql/pull/1639/commits/593ca574978de92385a8b4dce6e2d6d3b05a507e